yetanotherco/aligned_layer
# remove: deprecated verifyBatchInclusion function

> [!CAUTION]
> This PR modifies contracts, they should be upgraded before merge

>[!WARNING]
> This PR modifies contracts, they should be upgraded before merge

# Remove deprecated verifyBatchInclusion

## Description

This PR removes the deprecated `verifyBatchInclusion` function from the `AlignedLayerServiceManager.sol`

For this change, the AlignedLayerServiceManager contracts must be redeployed.

## To Test

- Test everything is running smoothly
- Deploy the old version of the contract in holesky, then run the upgrade. In the new version you will see only 1 `verifyBatchInclusion` function/

## Type of change


## Checklist

  - [Link to the open issue addressing it]()
